src/workflows-app/WorkflowsInWorkspace.ts

```diff
@@ -68,8 +68,14 @@ export const WorkflowsInWorkspace = ({
 
   const loadCbasInfo = useCallback(
     async (cbasProxyUrlDetails) => {
-      const { build } = await Cbas(signal).info(cbasProxyUrlDetails.state);
-      setCbasBuildInfo(build);
+      try {
+        const { build } = await Cbas(signal).info(cbasProxyUrlDetails.state);
+        setCbasBuildInfo(build);
+      } catch (error) {
+        notify('error', 'Error loading CBAS build information', {
```

Comment:
Just to clarify - if there is an error in fetching build info, the delete workflow button won't be available/disabled?

Reply:
yeah, it won't appear at all